pkg/log/pterm/section_logger.go

```diff
@@ -0,0 +1,135 @@
+package pterm
+
+import (
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/pterm/pterm"
+	"github.com/vmware-labs/distribution-tooling-for-helm/internal/widgets"
+	"github.com/vmware-labs/distribution-tooling-for-helm/pkg/log"
+)
+
+const (
+	// Number of spaces for each nesting level
+	nestSpacing = 3
+)
+
+// Logger defines a logger implemented using pterm
+type Logger struct {
+	writer io.Writer
+	level  log.Level
+	prefix string
+}
```

Comment:
Doesn't this and its methods belong in `logger.go` instead?

Reply:
Good catch. I was moving code around and i missed that.